lib/file-stream.js

```diff
@@ -92,11 +87,12 @@ class FileStream extends stream.Readable {
       this.push(buffer)
 
       if (this._missing === 0) this.push(null)
+      cb()
     })
     this._piece += 1
   }
 
-  _destroy (err, cb) {
+  _destroy (cb, err) {
```

Comment:
Why is this inverse?

Reply:
oh, reallistically err shouldnt be here

Reply:
no, error should be there, this is correct, in streamx the order is inversed

Reply:
see:
https://github.com/streamxorg/streamx/blob/master/index.js#L189
https://github.com/streamxorg/streamx/blob/master/index.js#L440

this is in case _destroy is called with an error without calling destroy(), which is... shouldnt be done, but since it was there for the original i kept it